exercises/diamond/example.rs

```diff
@@ -0,0 +1,63 @@
+static ABC: &'static str = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+pub fn get_diamond(c: char) -> Vec<String> {
+    if c == 'A' {
+        return vec![String::from("A")];
+    }
+    let mut result: Vec<String> = Vec::new();
+
+    //build first half
+    for e in ABC.chars() {
+        result.push(get_line(e, c).clone());
+        if e == c {
+            break;
+        }
+    }
+
+    //build second half
+    let mut rev = result.clone();
+    rev.pop(); //remove middle pice to avoid duplicates
+    for e in rev.iter().rev().into_iter() {
+        result.push((*e).clone());
+    }
+
+    result
+}
+
+fn get_line(e: char, c: char) -> String {
```

Comment:
It's not obvious to me reading this code what `e` and `c` are supposed to _mean_.

Here, it's fine; given that the code works, we can consider this to be more or less write-only code. If I were reviewing this at work, I'd demand better variable names, so that future maintainers will have a better chance of understanding it.